fabtests/include/hmem.h

```diff
@@ -35,6 +35,14 @@
 #include <rdma/fi_domain.h>
 #include <rdma/fi_errno.h>
 
+typedef enum {
+	CUDA_MEMORY_SUPPORT__NOT_INITIALIZED = -1,
+    CUDA_MEMORY_SUPPORT__NOT_SUPPORTED = 0,
+    CUDA_MEMORY_SUPPORT__DMA_BUF_ONLY = 1,
+    CUDA_MEMORY_SUPPORT__GDR_ONLY= 2, 
+    CUDA_MEMORY_SUPPORT__DMABUF_GDR_BOTH  = 3,
+} cuda_memory_support_e;
```

Comment:
Don't use typedef here. Just define the enum, and use `enum enum_name var_name` to define variables.

I would use `FT_CUDA_xxx` instead of `CUDA_xxx` to avoid name space confusion. So for the lowercase names.